server/controllers/LoanController.js

```diff
@@ -236,7 +236,24 @@ const loanController = {
     } else {
       statusMessageFunction(res, 400, `${loanStatus.badRequestMessage}` )
     }
-  }
-  
+  },
+  async allRepaymentLoan (req, res){
+    try{
+      const {rows} = await pool.query(queryTable.getAllRepayments)
```

Comment:
Who should be able to view the repayment history?

Reply:
Admin and user !?